stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss

```diff
@@ -119,24 +121,34 @@
   }
 }
 
-[dir="rtl"] {
-  header {
+[dir='rtl'] {
+  .global-header {
     .mid-nav {
       .menu {
-        &:first-child {
-          li {
-            margin: $spacing-12 4.4375rem $spacing-12 0;
 
-            &:first-child {
-              margin-right: 0;
-            }
+        &:last-child {
+          justify-content: flex-start;
+          padding-left: 0;
+          padding-right: 6.9375rem;
+        }
+  
+        li {
+          margin: $spacing-12 4.4375rem $spacing-12 0;
+          
+  &:first-child{margin-right: 0;}
```

Comment:
@Ashish-srij Kindly fix alignment and also format it properly.

Reply:
@shishirphadke Done please check.